src/renderer/components/+cluster-settings/components/cluster-icon-setting.tsx

```diff
@@ -28,7 +28,7 @@ export class ClusterIconSetting extends React.Component<Props> {
     try {
       if (file) {
         const buf = Buffer.from(await file.arrayBuffer());
-        cluster.preferences.icon = `data:image/jpeg;base64, ${buf.toString('base64')}`;
+        cluster.preferences.icon = `data:image;base64, ${buf.toString('base64')}`;
```

Comment:
Is it a valid mime-type to use as data-url?

Reply:
No thanks for the catch, I did a quick (and apparently dirty) sanity check which didn't find anything wrong with this. But looking closer it is wrong, so I will fix it.